tests/Properties.hs

```diff
@@ -536,5 +556,20 @@ misc_tests =
     , testProperty "readIntegerUnsafe" prop_readIntegerUnsafe
     ]
 
+strictness_checks =
+  [ testGroup "Lazy Word8"
+    [ testProperty "foldr is lazy"     prop_Lfoldr_lazy
```

Comment:
I think it is easier for readers, when properties are inlined under `testProperty`, especially if they are reasonably short, as in our case.

Reply:
It is all the same to me. Do you suggest that I inline the properties I add? I can if you like, but properties of no length are inlined anywhere in this file, so it would be inconsistent. The right way, as I see it, is to merge this pull request the way it is right now and then rearrange things in a separate boring pull request, as is customary. I do not care either way.

Reply:
Yes, I suggest to inline new properties, even while it makes style inconsistent, because it is (marginally) easier to review. This is a minor thing, feel free to ignore, if it compromises your values.

Reply:
I do not have any value judgement on this. Since it would be better for you, I shall fix it.